### ISSUE TITLE
Wrong indentation of C# fluent style function calls

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1278]
```
using System;
using System.Collections.Generic;
using System.Globalization;
using UnityEngine;
using UnityEngine.Serialization;
namespace UnityEngine.Experimental.Input
{
    public class ActionMap : ScriptableObject, IControlDomainSource
    {
        public List<InputControl> BuildControlsList()
        {
            ControlSetup controlsSetup = new ControlSetup();
            for (int i = 0; i < actions.Count; i++)
            {
                var action = actions[i];
                SupportedControl supportedControl = (SupportedControl)(typeof(SupportedControl)
                                                                       .GetMethod("Get")
                                                                       .MakeGenericMethod(actions[i].controlType)
                                                                       .Invoke(null, new object[] { actions[i].name }));
                action.controlIndex = controlsSetup.AddControl(supportedControl).index;
            }
            return controlsSetup.controls;
        }
    }
}


```

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60026-UNI-19895.cs" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60026-UNI-19895.cs"
index 1fedd58..377e3ea 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60026-UNI-19895.cs"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60026-UNI-19895.cs"
@@ -1,26 +1,26 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
 using UnityEngine;
 using UnityEngine.Serialization;
 namespace UnityEngine.Experimental.Input
 {
     public class ActionMap : ScriptableObject, IControlDomainSource
     {
         public List<InputControl> BuildControlsList()
         {
             ControlSetup controlsSetup = new ControlSetup();
             for (int i = 0; i < actions.Count; i++)
             {
                 var action = actions[i];
 // This line is kept with 71 spaces.
                 SupportedControl supportedControl = (SupportedControl)(typeof(SupportedControl)
-                    .GetMethod("Get")
-                    .MakeGenericMethod(actions[i].controlType)
-                    .Invoke(null, new object[] { actions[i].name }));
+                                                                       .GetMethod("Get")
+                                                                       .MakeGenericMethod(actions[i].controlType)
+                                                                       .Invoke(null, new object[] { actions[i].name }));
                 action.controlIndex = controlsSetup.AddControl(supportedControl).index;
             }
             return controlsSetup.controls;
         }
     }
 }
```

Debug support files:
---
[60026-UNI-19895.cs](https://gist.githubusercontent.com/mihaipopescu/616895b9bf3d8ab2aef7671798501756/raw/1fedd5807aa6a7cf49dda6992192078628c578f3/60026-UNI-19895.cs)
[60026-UNI-19895.cs.log](https://gist.githubusercontent.com/mihaipopescu/616895b9bf3d8ab2aef7671798501756/raw/612925da6485d176dbb253b3a8a2a51bf5b70cd0/60026-UNI-19895.cs.log)
[60026-UNI-19895.cs.unc](https://gist.githubusercontent.com/mihaipopescu/616895b9bf3d8ab2aef7671798501756/raw/6260595c07ec4dfebaeaca7481253378ea1f85ed/60026-UNI-19895.cs.unc)
[Uncrustify.CSharp.cfg](https://gist.githubusercontent.com/mihaipopescu/616895b9bf3d8ab2aef7671798501756/raw/1d064c1fabb76ce328dd015ed40023207de20ca1/Uncrustify.CSharp.cfg)
[UNI-19895.cs](https://gist.githubusercontent.com/mihaipopescu/616895b9bf3d8ab2aef7671798501756/raw/a22c6fa86eff6e2fe353a14528f440d7cf270c5e/UNI-19895.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
I get this output:
```cpp
using System;
using System.Collections.Generic;
using System.Globalization;
using UnityEngine;
using UnityEngine.Serialization;
namespace UnityEngine.Experimental.Input
{
    public class ActionMap : ScriptableObject, IControlDomainSource
    {
        public List<InputControl> BuildControlsList()
        {
            ControlSetup controlsSetup = new ControlSetup();
            for (int i = 0; i < actions.Count; i++)
            {
                var action = actions[i];
// This line is kept with 71 spaces.
                SupportedControl supportedControl = (SupportedControl)(typeof(SupportedControl)
                                                                       .GetMethod("Get")
                                                                       .MakeGenericMethod(actions[i].controlType)
                                                                       .Invoke(null, new object[] { actions[i].name }));
                action.controlIndex = controlsSetup.AddControl(supportedControl).index;
            }
            return controlsSetup.controls;
        }
    }
}
```

**Comment by mihaipopescu** [comment]
Yes, and we expect to have just one tab of indentation.

**Comment by gmaurel** [comment]
Fixed at f2285d9a1852d4d1c59399256a904497a7401f7c, Dec 14 18:26:11 2017
